api/v1/groupversion_info.go

```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 
 // Package v1 contains API Schema definitions for the tools v1 API group
-//+kubebuilder:object:generate=true
-//+groupName=tools.opdev.io
```

Comment:
I believe the lack of a space here was important, given that it's a marker (vs. a comment). Was this added by a linter, or is this being added to be consistent with newer generators?

Reply:
I believe that the new tooling versions is doing this, though it might be my IDE, so I'll re-run outside of the IDE.

Reply:
Looks like it doesn't matter either way.

Reply:
It looks like running `make` cmds changes this as well. So we can leave it.